Model/Method/Capayable.php

```diff
@@ -107,6 +107,7 @@ public function __construct(
         ConfigProviderFactory $configProviderFactory = null,
         ConfigProviderMethodFactory $configProviderMethodFactory = null,
         PricingHelperData $priceHelper = null,
+        \Buckaroo\Magento2\Model\SecondChanceRepository $secondChanceRepository,
```

Comment:
use class alias (so that is consistent with the other constructor parameters)

Reply:
Thank you, fixed